data-reporting-service/investigation-service/src/main/java/gov/cdc/etldatapipeline/investigation/service/InvestigationService.java

```diff
@@ -57,20 +59,24 @@ public String processInvestigation(String value) {
             JsonNode payloadNode = jsonNode.get("payload").path("after");
             if (payloadNode != null && payloadNode.has("public_health_case_uid")) {
                 String publicHealthCaseUid = payloadNode.get("public_health_case_uid").asText();
+
+                // Calling sp_public_health_case_fact_datamart_event
+                logger.info("Executing stored proc with ids: {} to populate PHS fact datamart", publicHealthCaseUid);
+                investigationRepository.populatePhcFact(publicHealthCaseUid);
+                logger.info("Stored proc executed");
+
                 logger.debug(topicDebugLog, publicHealthCaseUid, investigationTopic);
                 Optional<Investigation> investigationData = investigationRepository.computeInvestigations(publicHealthCaseUid);
                 if(investigationData.isPresent()) {
                     InvestigationTransformed investigationTransformed = processDataUtil.transformInvestigationData(investigationData.get());
                     kafkaTemplate.send(investigationTopicOutputTransformed, investigationTransformed.toString());
-                    return objectMapper.writeValueAsString(investigationData);
+                    return objectMapper.writeValueAsString(investigationData.get());
                 }
             }
+
         } catch (Exception e) {
             logger.error("Error processing investigation: {}", e.getMessage());
```

Comment:
@sveselev how feasible is it to also include the list of Ids that are passed with the error message.?